scripts/windows/openclaude-aliases.ps1

```
@@ -0,0 +1,251 @@
+    [string]$Provider,
+    [ValidateSet("latency", "balanced", "coding")]
+    [string]$Goal = "coding",
+    [string]$Model,
+    [string]$ApiKey
+  )
+
+  Assert-OpenClaudeCommand -Name "bun" -InstallHint "Install Bun from https://bun.sh and open a new terminal."
+
+  Push-Location (Get-OpenClaudeRepoRoot)
+  try {
+    $args = @("run", "profile:init", "--", "--provider", $Provider)
```

PSScriptAnalyzer (1.25.0)

[warning] The Variable 'args' is an automatic variable that is built into PowerShell, assigning to it might have undesired side effects. If assignment is not by design, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)